src/lsm/table_memory.zig

```diff
@@ -804,17 +838,29 @@ test "table_memory (secondary): annhiliation yields zero after deduplicate" {
 
     const alloc = testing.allocator;
 
+    var radix_scratch = RadixBuffer{
+        .buffer = try alloc.alignedAlloc(
+            u8,
+            @alignOf(Value),
+            @sizeOf(Value) * Table.value_count_max,
+        ),
+        .in_use = false,
+    };
+    defer alloc.free(@as([]align(@alignOf(Value)) u8, @alignCast(radix_scratch.buffer)));
```

Comment:
This is a very specific code smell! These lines are related three times in tests, but only once in production code. This is baad, because it makes refactoring disproportionally painful. For example, to introduce init/deinit, you need to change prod code only once, but there are there test places to fix. 

So, it's better to proactively extract such common setup-code into a fixture, something like `TestHelper.create_scratch_buffer` (see also `fixtures.zig`). Though, if you refactor that to init/deinit, it'll be only two lines, so maybe not worth extracting!